### PR TITLE
Do Not Merge -- More physics

### PR DESCRIPTION
**Pull request by sethalves**
experiments / tuning on physics

### CODE REVIEW

File: libraries/physics/src/EntityMotionState.cpp [hunk 1]
```diff
@@ -151,10 +151,10 @@ void EntityMotionState::setWorldTransform(const btTransform& worldTrans) {
...
     }
 
-    uint32_t ownershipClaimDelay = 50; // TODO -- how to pick this?  based on meters from our characterController?
+    uint32_t ownershipClaimDelay = 180; // TODO -- how to pick this?  based on meters from our characterController?
 
     if (_movingStepsWithoutSimulationOwner > ownershipClaimDelay) {
-        //qDebug() << "Warning -- claiming something I saw moving." << getName();
+        qDebug() << "Warning -- claiming something I saw moving." << getName();
         setShouldClaimSimulationOwnership(true);
     }
```
Comment (AndrewMeadows) [inline libraries/physics/src/EntityMotionState.cpp]: I don't think we should merge this line uncommented out (*).  Better to manually uncomment it when debugging this problem.

(*) Very verbose logging kills performance on windows.

**Comment by hifi-gustavo** [comment]
Build console output for commit: ee98c9534766a1fdef05af65ceacd7955dc21ea9 available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4856ebdb9e795659dc80a1913ddd59d9d93ff8ba available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: af939fbfc47dacc720d78fd727234b37e5348494 available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 96d69df16e619f57d7ae64e9d9a6dba4a5b75e9a available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 26eeabe330a97b0ac92cf0b1109e178148d51aad available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5373d80f6c95f35c46f49504faccbe8d00b36492 available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 897bfcf0ef25d8a2114b09ec64b2a8972a3c446c available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 23aacc56f76180ea31cbfe5a9efe4db7cd9715f1 available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c824bb06190205603e53451be303a52dda93a40e available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 998a66f715e1de21d9bff396cfd316bea9658068 available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2dca015fafaf3422d0ce4e27efefab1e57ab3bdb available at: https://gist.github.com/3c1bad84d84e3f252a77<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4810.dmg

**Pull request closed without merging**